src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj

```diff
@@ -1,57 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
(I may also have misunderstood what's going on here - there are a number of things changing such as "fsc-proto.exe" --> "fsc.exe" and it's possible that relatively simple paths to a Mono-only and .NET-Fx-only builds are still open, or at least no further away)